prometheuspvesd/discovery.py

```diff
@@ -157,6 +157,11 @@ def _exclude(self, pve_list):
             if str(obj["vmid"]) in self.config.config["exclude_vmid"]:
                 continue
 
+            if isinstance(obj["tags"], str) \
+                    and not set(obj["tags"].split(","))\
+                    .isdisjoint(map(str, self.config.config["exclude_tags"])):
```

Comment:
Lists can be passed to `isdisjoint` as well, and `exclude_tags` is ensured to be a list of strings by the JSON schema already.  Is there another reason why we need the `map` here?

Besides that, I would prefer to format multiline statements this way:

```
if isinstance(obj["tags"], str) and not set(obj["tags"].split(",")).isdisjoint(
    map(str, self.config.config["exclude_tags"])
):
    continue
```